discovery-provider/src/queries/notifications.py

```diff
@@ -726,6 +731,66 @@ def notifications():
             track_id = entry.track_id
             owner_info[const.tracks][track_id] = owner
 
+        # Get playlist updates
+        today = date.today()
+        thirty_days_ago = today - timedelta(days=30)
+        thirty_days_ago_time = datetime(
+            thirty_days_ago.year,
+            thirty_days_ago.month,
+            thirty_days_ago.day,
+            0,
+            0,
+            0
+        )
+        playlist_update_query = session.query(Playlist)
+        playlist_update_query = playlist_update_query.filter(
+            Playlist.is_current == True,
+            Playlist.is_delete == False,
+            Playlist.blocknumber > min_block_number,
+            Playlist.blocknumber <= max_block_number)
+
+        playlist_update_results = playlist_update_query.all()
```

Comment:
Hmmmm, I don't know if this is the cleanest solution based on how the notifications endpoint was designed to work. It was created with the intent to be time-invariant - such that when requesting notifications from a min and max blocknumber - the same notifications are always returned. Seeing as this is 30 days though and we don't need to catch up it seems fine - but I think it would be technically more correct if the endpoint handler took in the date as a query param. Thoughts @hareeshnagaraj - I think this should be fine though.

Reply:
Yeah good callout @jowlee this is one of the items I was going to surface with the team.
I believe that with this 30 days and the blocknumbers in the query, we may miss a good number of notifications. This is the most important piece I'm looking at in this PR to make sure the logic works.